Resources/xslt-stylesheets/rezeptanforderung.xsl

```diff
@@ -0,0 +1,1011 @@
+                                    </xsl:when>
+                                    <xsl:when
+                                    test="$rootBundle/fhir:entry/fhir:resource/fhir:MessageHeader/fhir:eventCoding/fhir:code/@value = 'eRezept_Rezeptanforderung;Rezeptanfrage_Ablehnung'">
+                                        <xsl:text>Kopie einer Ablehnung einer Rezeptanforderung</xsl:text>
+                                    </xsl:when>
+                                    <xsl:when
+                                    test="$rootBundle/fhir:entry/fhir:resource/fhir:MessageHeader/fhir:eventCoding/fhir:code/@value = 'eRezept_Rezeptanforderung;Rezeptbestaetigung'">
+                                        <xsl:text>Kopie einer Rezeptbestätigung</xsl:text>
+                                    </xsl:when>
+                                </xsl:choose>
+                            </xsl:when>
+                            <!-- Default subject if no matching code is found -->
+                            <xsl:otherwise>
+                                <xsl:text>Unknown Subject</xsl:text>
+                            </xsl:otherwise>
+                        </xsl:choose>
+                    </h2>
+                </div>
+                <xsl:if
+                test="fhir:eventCoding/fhir:code/@value = 'eRezept_Rezeptanforderung;NachrichtKopie'">
+                    <div class="copy-sender-info clear">
+                        <p>Dies ist die Kopie einer initialen Nachricht, gesendet von:</p>
+                        <xsl:call-template name="sender-info">
+                            <xsl:with-param name="rootBundle" select="$rootBundle" />
+                        </xsl:call-template>
+                        <br />
+                        <p>an: </p>
+                        <br />
+                        <xsl:call-template name="receiver-info">
+                            <xsl:with-param name="rootBundle" select="$rootBundle" />
+                        </xsl:call-template>
+                    </div>
+                </xsl:if>
+            </xsl:for-each>
+        </div>
+    </xsl:template>
+    <!-- Adjust service-request-info template to use rootBundle -->
+    <xsl:template name="service-request-info">
+        <xsl:param name="patientRef" />
+        <xsl:param name="rootBundle" />
+        <div class="service-request-info">
+            <xsl:call-template
+            name="sr-patient-info">
+                <xsl:with-param name="patientRef" select="$patientRef" />
+                <xsl:with-param name="rootBundle" select="$rootBundle" />
+            </xsl:call-template>
+            <!-- Template for displaying information about each ServiceRequest in the Bundle -->
+            <div
+            class="service-request">
+                <xsl:for-each
+                select="$rootBundle/fhir:entry/fhir:resource/fhir:ServiceRequest[fhir:subject/fhir:reference/@value = concat('Patient/', $patientRef)]">
```

Comment:
Für den Fall das die Referenzen per "uuid:" Präfix und nicht mit "Patient/"-Prefix referenziert sind, bricht die Darstellung

Reply:
Das würde ich in der kontinuierlichen Weiterentwicklung verbessern.